doc/tutorials/using_ubuntu_as_sos.rst

```diff
@@ -0,0 +1,262 @@
+.. _Ubuntu Service OS:
+
+Using Ubuntu as the Service OS
+##############################
+
+This document explains how to use Ubuntu as the Service OS with the ACRN hypervisor. This document
+builds on the :ref:`getting_started`. The Service OS used as a reference in Project ACRN is based
+on `Clear Linux`_. This document explains how to use Ubuntu instead, the User OS is the same one as described
+in the :ref:`getting_started`. The hardware platform used throughout this document is the `Intel NUC Kit
+NUC6CAYH <https://www.intel.com/content/www/us/en/products/boards-kits/nuc/kits/nuc6cayh.html>`_.
+
+Install Ubuntu (natively)
+*************************
+
+Ubuntu 16.04.4 LTS was used throughout this document, other versions such as 18.04 may work too.
```

Comment:
Unless you've tested it, I'd leave off mentioning 18.04.

Reply:
I want that we do a final run through these instructions (me or someone else), we can settle on one of those at that point.